### ISSUE TITLE
npm run-script release:mac success, but cannot be opened because of a problem.

### ISSUE THREAD

**Issue by hitzhangjie** [marktext/marktext#527]
Hi, I use marktext as my Markdown editor, I like it, I want to make some changes to it to meet my needs, so I download the source code and make some changes to it. I run `npm run-script dev` to test the changes made to it, it works, so I want to install it to /Applications.

After that, I run command `npm run-script release:mac` to build the release version *.dmg on my mac. It successfully build the *.dmg install package, then I install it to /Applications. When I run it, it pops up the following dialog, it says 'Mark Text cannot be opened because of problem'.

![image](https://user-images.githubusercontent.com/3725760/47613737-7a50c800-dacf-11e8-8300-8454cd9331d9.png)

Please help, thanks in advance!

**Comment by hitzhangjie** [comment]
I see some crash logs about electron dynamic lib, I installed electron with `npm install -g electron --save-dev`, I don't know whether it is relevant with this problem.

**Comment by hitzhangjie** [comment]
Ah, I uninstall electron with `npm rm -g electron`, then I rebuilt Mark Text with `npm run-script release:mac`, the new built install package works!

**Comment by Jocs** [comment]
@hitzhangjie At last, you fix it, and build successfully?

**Comment by hitzhangjie** [comment]
Actually, I just changed the `right-column right border style`, make it display forever, when open a project, the length of different files' name are different, it looks like a little messy, so I make the right border display for ever. Users can position the mouse cursor to drag to adjust the sidebar width more conveniently.

And, the icon to `open folder`, I changed the svg icon from `icon-create-project` to `icon-folder-open`, because the former one looks like create a new document rather than open folder.
 
![image](https://user-images.githubusercontent.com/3725760/47613897-1085ed80-dad2-11e8-97df-bce77cd8fbdd.png)
